src/main/java/net/sf/mzmine/modules/visualization/peaklisttable/PeakListTablePopupMenu.java

```diff
@@ -33,7 +33,6 @@
 import javax.swing.ListSelectionModel;
 import javax.swing.SwingUtilities;
 import javax.swing.table.AbstractTableModel;
-import com.google.common.collect.Range;
```

Comment:
Removing this import causes compilation to fail. Why did you remove it?

Reply:
Thats weird. I did not intentionally delete it. Maybe the auto import module in eclipse accidentally deleted it.
Sorry I thought I only did two small changes.
Can you change and reimport it?
I could do it tomorrow.

Thanks for your review. And sorry for that problem.